daq/base_gateway.py

```diff
@@ -224,6 +229,49 @@ def terminate(self):
                 LOGGER.error('Gateway %s terminating dummy: %s', self.name, e)
                 LOGGER.exception(e)
 
+    def _get_scan_interface(self):
+        return self.host, self.host_intf
+
+    def discover_host(self, subnets: List[ip_network], mac: str, callback: Callable, vid=None) -> ip_address:
```

Comment:
is this discover_host or discover_hosts (plural)?  Also, does the -> ip_address at the end imply that it's returning an ip_address... ?  Does it actually return anything (seems like it's callback-oriented, not return-oriented...).

Reply:
If this is in fact discover_host (as I think it is after I looked at it), then the mac argument should likely be before the subnets argument (which is what caused some of my confusion).